ibm/mas_devops/roles/cp4d_admin_pwd_update/tasks/main.yml

```diff
@@ -0,0 +1,92 @@
+# 1. Provide debug information to the user
+---
+- name: "Debug information for changing CP4D admin password"
+  debug:
+    msg:
+      - "cp4d_namespace .............. {{ cp4d_namespace }}"
```

Comment:
Style guidelines ... can you align these types of debug logging ... https://github.com/ibm-mas/ansible-devops/blob/master/CONTRIBUTING.md#align-debug-messages